include/ica_api.h

```diff
@@ -3458,6 +3464,32 @@ ICA_EXPORT
 int ica_aes_gcm_kma_verify_tag(const unsigned char* known_tag, unsigned int tag_length,
 		const kma_ctx* ctx);
 
+/**
+ * Obtain the iv from the given context. This function is mainly intended to
+ * allow applications to query an internally created iv when running in fips
```

Comment:
also here, one can ALWAYS obtain the IV, not only when in FIPS mode. Maybe rephrase that the IV can be obtained when it was not specified during ica_aes_gcm_kma_init()

Reply:
Yep, same as above, will change the text